app/code/community/Rakuten/RakutenPay/Helper/Data.php

```diff
@@ -64,6 +64,8 @@ public function __construct()
     private function environmentNotification()
     {
         \RakutenPay\Resources\Log\Logger::info('Processing environmentNotification.');
+        \RakutenPay\Resources\Log\Environment::infoVersions();

```

Comment:
A better name to convey a real meaning of this functions would be:

```php
  \RakutenPay\Resources\Log\Environment::logInfoVersions();
  \RakutenPay\Resources\Log\Environment::logInfoPHPConfiguration();
```

Reply:
Yes! ;)